examples/intkey_go/src/sawtooth_intkey_client/utils.go

```diff
@@ -0,0 +1,61 @@
+/**
```

Comment:
this file should not be added at all (it is removed in the next commit) Please remove this from this commit.